pkg/controller/pcideviceclaim/pcideviceclaim_controller.go

```diff
@@ -335,45 +290,139 @@ func (h Handler) reconcilePCIDeviceClaims(name string, pdc *v1beta1.PCIDeviceCla
 		return pdc, nil
 	}
 
-	if pdc.Spec.NodeName == h.nodeName && !pdc.Status.PassthroughEnabled {
-		newPdc, err := h.attemptToEnablePassthrough(pdc)
-		return newPdc, err
+	if pdc.Spec.NodeName != h.nodeName || pdc.Status.PassthroughEnabled {
```

Comment:
currently if a pcideviceclaim is already enabled, then we dont start the plugin due to this check.

Reply:
Fixed in a new commit. I moved the node check to the beginning of the reconcile function, then only returned if `PDC.passthroughEnabled && dp != nil`